tests/util/hash.py

```diff
@@ -8,9 +9,20 @@ def unordered(file_path):
     """
     file_hash = 1
     eighth_mersenne_prime = 2147483647
-    with open(file_path) as file:
+    with open(file_path, errors="ignore") as file:
```

Comment:
Out of curiosity, why `errors="ignore"`?

Reply:
This is to prevent unicode errors from happening when hashing non-ASCII encoded files. However, it isn't being actively used (since binary files are being hashed with MD5), so I can remove it.

Reply:
Defs remove if unnecessary